x-pack/plugins/triggers_actions_ui/public/common/expression_items/threshold.tsx

```diff
@@ -152,7 +153,7 @@ export const ThresholdExpression = ({
                       data-test-subj="alertThresholdInput"
                       min={0}
                       value={!threshold || threshold[i] === undefined ? '' : threshold[i]}
-                      isInvalid={errors[`threshold${i}`]?.length > 0 || !threshold[i]}
+                      isInvalid={errors[`threshold${i}`]?.length > 0 || isNil(threshold[i])}
```

Comment:
isNil should be applied to line 149 as well :)

Reply:
Oh yes, good catch! Thanks!

Reply:
Resolved in https://github.com/elastic/kibana/pull/142225/commits/29c3856273a60e5fd4de53f195aab73a40167c06